auth_backends/backends.py

```diff
@@ -59,10 +86,43 @@ def auth_complete(self, *args, **kwargs):
         self.auth_complete_signal.send(sender=self.__class__, user=user, id_token=self.id_token)
         return user
 
+    def validate_and_return_id_token(self, id_token):
+        """
+        Validates the id_token according to the steps at
+        http://openid.net/specs/openid-connect-core-1_0.html#IDTokenValidation.
+        """
+        id_token = self._jwt_decode(id_token)
+
+        # Verify the token was issued in the last 10 minutes
+        utc_timestamp = timegm(datetime.datetime.utcnow().utctimetuple())
+        if id_token['iat'] < (utc_timestamp - 600):
```

Comment:
This value should not be hard-coded. It should be configurable (via setting) for the provider.